components/security/audit/frontend/src/main/resources/PhenoTips/AuditResults.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+-->
+
+<xwikidoc version="1.1">
+  <web>PhenoTips</web>
+  <name>AuditResults</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1401822196000</creationDate>
+  <parent>PhenoTips.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <date>1401822196000</date>
+  <contentUpdateDate>1401822196000</contentUpdateDate>
+  <version>1.1</version>
+  <title/>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>{{velocity wiki="false"}}
+$response.setContentType('application/json')
+#set ($dateFormat = "yyyy/mm/dd hh:mm")
+#set($map = {})
+#set($rows = [])
+#set($offset = $mathtool.toInteger($request.get('offset')))
+#set($limit = $mathtool.toInteger($request.get('limit')))
+#if (!$limit)
+  #set ($limit = 25)
+#end
+#set($user = $request.get('user'))
+#set($ip = $request.get('ip'))
+#set($action = $request.get('action'))
+#set($entity = $request.get('record'))
+#set($fromTime = $request.get('fromTime'))
+#set($toTime = $request.get('toTime'))
+#set($data = $services.audit.getEvents($offset, $limit, $action, $user, $ip, $entity, $fromTime, $toTime))
```

Comment:
Bug: the java code is 0-indexed, but the livetable is 1-indexed. The first event is always inaccessible.